src/components/Reports/Subscribers/SubscribersRedirect.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { InjectAppServices } from '../../../services/pure-di';
+import { useParams, Redirect } from 'react-router-dom';
+import SafeRedirect from '../../SafeRedirect';
+import { replaceSpaceWithSigns, extractParameter } from '../../../utils';
+import queryString from 'query-string';
+
+const SubscribersRedirect = ({ location }) => {
```

Comment:
What do you think about renaming the component as something that denotes the patch? for example: `SubscribersLegacyUrlRedirect` or something like that.